release/release.go

```diff
@@ -109,8 +111,36 @@ var fileExists = func(path string) bool {
 	return err == nil
 }
 
-func isWSL() bool {
-	return fileExists("/proc/sys/fs/binfmt_misc/WSLInterop")
+var filesystemRootType = func() (int64, error) {
```

Comment:
it would be good to have a test for this itself, maybe comparing result with what we get from `stat -f -c "%t" /`